site/src/pages/components/invokers.explainer.mdx

```diff
@@ -110,41 +107,42 @@ some cases, see
 [the popovertarget attr-asociated element steps](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#attr-associated-element)
 for more).
 
-The `command` (and the `.command` reflected property) is a freeform hint to the
-invoker target. Command can be a "built-in" action or a "custom" action. If
-`command` is not supplied then it is considered invalid and no action will take
-place, and no event will be fired.
+The `command` attribute (and the `.command` reflected property) is a freeform
+hint to the invoker target. Command can be a "built-in" action or a "custom"
+action. If `command` is not supplied then it is considered invalid and no
+action will take place, and no event will be fired.
 
-Custom command values _must_ contain a `-`. They will _never_ trigger a browser
-default behaviour, aside from dispatching an `CommandEvent`. This allows web
-developers to create custom Invoke actions for their components.
+Custom command values _must_ start with a `-`. This is a safe namespace to use,
```

Comment:
Did you mean "contain a `-`"? I don't know of any other namespace in HTML that uses a single dash `-` for custom things. Custom elements must just **contain** a `-`.